### DISCUSSION TITLE
我的博客没有出现在成员列表

### DISCUSSION THREAD

**Discussion by minglinxuan** [travellings-link/travellings#954] [寻求帮助] [answered]
emm 忘记多久之前了 通知我说已经加入本计划
而本站也挂上了计划地址 但最近在成员列表没有看见我的博客...
My blog：blog.iqia.top

**Comment by minglinxuan** [comment]
![微信图片_20220422222856](https://user-images.githubusercontent.com/77839269/164734977-5e829597-5fde-4b8c-a4db-223b1f6fac93.png)
Look at this

**Comment by linlinzzo** [comment]
https://github.com/travellings-link/travellings/issues/703 这个吗？麻烦您重开一个申请 issue，连接到这个 discussion 说明一下问题。

**Comment by linlinzzo** [comment]
我在项目群里说明了，请等待处理。（突然发现你开不了空白 issue）

**Comment by imjason** [comment]
这个我们将会尽快处理，我们自己的工作也比较繁重，希望理解

**Comment by HAHH9527** [comment, marked as answer]
检查发现已正确添加
